fe/fe-core/src/main/java/org/apache/doris/transaction/PublishVersionDaemon.java

```diff
@@ -240,7 +241,13 @@ private void publishVersion() throws UserException {
                     // one transaction exception should not affect other transaction
                     globalTransactionMgr.finishTransaction(transactionState.getDbId(), transactionState.getTransactionId(), publishErrorReplicaIds);
                 } catch (Exception e) {
-                    LOG.warn("error happends when finish transaction {} ", transactionState.getTransactionId(), e);
+                    if (e instanceof MetaNotFoundException) {
+                        globalTransactionMgr.abortTransaction(transactionState.getDbId(), transactionState.getTransactionId(), e.getMessage(),
```

Comment:
Do not need abort txn here? 
If the database is force dropped, we could not get db related txn manager and will skip the txn in next loop.